CalibMuon/DTCalibration/test/DBTools/DumpDBToFile.cc

```diff
@@ -98,15 +96,9 @@ void DumpDBToFile::beginRun(const edm::Run&, const EventSetup& setup) {
     setup.get<DTReadOutMappingRcd>().get(channels);
     channelsMap = &*channels;
   } else if (dbToDump == "RecoUncertDB") {
-    if (format == "Legacy") {
-      ESHandle<DTRecoUncertainties> uncerts;
-      setup.get<DTRecoUncertaintiesRcd>().get(uncerts);
-      uncertMap = &*uncerts;
-    } else if (format == "DTRecoConditions") {
-      ESHandle<DTRecoConditions> h_rconds;
-      setup.get<DTRecoConditionsUncertRcd>().get(h_rconds);
-      rconds = &*h_rconds;
-    }
+    ESHandle<DTRecoConditions> h_rconds;
+    setup.get<DTRecoConditionsUncertRcd>().get(h_rconds);
+    rconds = &*h_rconds;
```

Comment:
Hi @namapane since you are touching this part of the code, can you please move to the new method with the Tokens?
To use ESGetToken see
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#In_ED_module
To get data with the token see
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#Getting_data_from_EventSetup_wit

Reply:
OK but I may need to change the subject of the PR, since that will amount tomore changes than those targeted here :-)

Reply:
I'm also fine if it's done in a separate PR if you prefer that.

Reply:
That would be great (I started coding but I'd need a moment to test the change for all 7 objects that are consumed here)